packages/d/devedeng/package.yml

```diff
@@ -12,9 +13,9 @@ rundeps    :
     - cdrtools
     - dvdauthor
     - libgtk-3
+    - python-gobject
     - python-urllib3
     - python3-cairo
```

Comment:
Just a minor thing. Please remove `python3-cairo` from rundeps list because it a rundep of `python-gobject`.